src/examples/cancel/Shootist.java

```diff
@@ -56,18 +76,12 @@
 
     private boolean requestTerminated;
 
-    private static Logger logger = Logger.getLogger(Shootist.class);
-
-
-
-
-
     public void processRequest(RequestEvent requestReceivedEvent) {
         Request request = requestReceivedEvent.getRequest();
         ServerTransaction serverTransactionId = requestReceivedEvent
                 .getServerTransaction();
 
-        logger.info("\n\nRequest " + request.getMethod()
+        LOG.info("\n\nRequest " + request.getMethod()
```

Comment:
A private static final field should be cased like that. I know this spreads through the test classes and I can't follow your argument concerning the log4j API.